sentry-types/src/protocol/attachment.rs

```diff
@@ -37,7 +39,7 @@ impl AttachmentType {
     }
 }
 
-#[derive(Clone, PartialEq, Default)]
+#[derive(Clone, PartialEq, Deserialize, Default)]
```

Comment:
I don’t think we use the `Deserialize` directly, since attachments have their own deserialize logic.